src/mrvi/_model.py

```diff
@@ -878,22 +990,41 @@ def perform_multivariate_analysis(
             a smaller set of cells to analyze, e.g., by specifying `adata`.
         store_lfc_metadata_subset
             Specifies a subset of metadata for which log-fold changes are computed.
-            These keys must be a subset of `donor_keys`.
+            These keys must be a subset of `sample_cov_keys`.
             Only applies when `store_lfc=True`.
         store_baseline
             Whether to store the expression in the module if logfoldchanges are computed.
         eps_lfc
             Epsilon to add to the log-fold changes to avoid detecting genes with low expression.
-        filter_donors
-            Whether to filter out-of-distribution donors prior to performing the analysis.
-        filter_donors_kwargs
+        filter_inadmissible_samples
+            Whether to filter out-of-distribution samples prior to performing the analysis.
+        filter_inadmissible_samples_kwargs
```

Comment:
```suggestion
        filter_samples_kwargs
```
Should be the last arg in the docstring